app/controllers/application_controller.rb

```diff
@@ -1,6 +1,5 @@
 class ApplicationController < ActionController::Base
     def hello
-        result = `python3 PythonScripts/crawlerSample.py Bolsonaro`
-        render html: 'hello, world!' + result
+        render html: 'Hello! Welcome to WHISPR, a place where you will find recent AND trustful contents about your favourite and most searched news. Happy News! Literally..'
```

Comment:
Mandando a true, ia ficar melhor um assim:
```suggestion
        render html: 'Hello! Welcome to WHISPR, a place where you will find recent AND trustful contents about your favourite and most searched news. Good News, everyone! Literally..'
```